examples/inverse/plot_lcmv_beamformer_volume.py

```diff
@@ -87,12 +88,20 @@
 
 clim = dict(kind='value', pos_lims=[0.3, 0.6, 0.9])
 stc.plot(src=forward['src'], subject='sample', subjects_dir=subjects_dir,
-         clim=clim)
+         clim=clim, verbose=True)
 
 ###############################################################################
-# We can also visualize the activity on a "glass brain" (shown here with
-# absolute values):
+# Now let's:
+# - visualize the activity on a "glass brain",
+# - show absolute values, and
+# - morph data to ``'fsaverage'`` instead of `sample`
 
+morph = mne.compute_source_morph(
+    forward['src'], 'sample', 'fsaverage', verbose=True)
```

Comment:
you should pass `subjects_dir` here, otherwise it can fail. It's happening for me.

Reply:
Not related to this PR, but there is a `_check_dep` at L144 of `morph.py` and then another one at `L691`. The one at L691 is stricter than the one at L144 but I'm not sure why ... they are both on the same code path